docs/source/roadmap.md

```diff
@@ -25,80 +25,8 @@ By enabling the creation of standardized, highly compressed outputs that are com
-3. Output formats can be loaded and are performant
-4. Processed data can be used to recapitulate results
-
-### _starfish_ output formats enable external annotation to place the data in anatomical and cell type context and allow comparison of across methods
-1. Processed Experiments (all FOVs for a given dataset) can produce Cell x Gene tables with each cell in the physical coordinates of the sample
-2. 2D polygon annotations of anatomical structures in physical coordinates can be saved in a standardized format and membership of each cell in annotation structures can be added as metadata to the Cell x Gene table
-3. Cell x Gene table can be easily imported by computational biologists (outside of _starfish_) to determine cell type or cluster membership for each cell.  This cluster membership can include probabilistic membership to multiple clusters and is added to the Cell x Gene table as metadata
-
-### A tool to convert image data into SpaceTx Format
-1. CLI and API (Java) can parse [supported file formats](https://docs.openmicroscopy.org/bio-formats/6.0.0-m3/supported-formats.html) into SpaceTx-formatted datasets<sup>[5](#fn5)</sup>
-2. Installation of the tool can be accomplished by downloading an archive with a launcher script or by using pre-built docker images
-3. Users and/or data wranglers can point the tool at one of the [stated files](https://docs.openmicroscopy.org/bio-formats/6.0.0-m3/formats/dataset-table.html) for each FOV to generate 2D TIFFs as well as the necessary SpaceTX JSON files
-4. In the case that multiple independent images (a “series”) are present in a single dataset, the user must specify which offset to use
-5. Multiple datasets can be passed in which case each in order is considered a separate field of view
-5. The starting index of the field of view may be changed from the default of zero
-6. If a codebook exists at the time of creation, it can be passed to the tool for inclusion in the JSON. Otherwise, a dummy codebook will be created
-
-## What we need to build: _starfish_ 0.1.0
-In general, we aim for _starfish_ to be a comprehensive platform for the processing of image-based transcriptomics experiments. It should be intuitive to use, fast, and should not require extensive experience with python programming. However, at least during the initial development period, _starfish_ will require domain experience in image processing, as our target users are experts in image generation and processing.
+## What is _starfish_?
+We aim for _starfish_ to be a comprehensive platform for the processing of image-based transcriptomics experiments. It should be intuitive to use, fast, and should not require extensive experience with python programming. However, at least during the initial development period, _starfish_ will require domain experience in image processing, as our target users are experts in image generation and processing.
 
 ### A General Format and Object Model for Image-based Transcriptomics
```

Comment:
We need to be _very_ clear in the paragraph below that pre-registration is a _current_ requirement, but not a future one.

Reply:
I consider everything post-MVP to be negotiable

Reply:
OK, reading this more closely in context, I see what you mean. Adding some text to clarify, but I want to return to this once we merge it in.